debian/landscape-client.postinst

```diff
@@ -106,6 +106,15 @@ END
         USER_UPDATE_FLAG_FILE="$DATA_PATH/user-update-flag"
         install --owner=landscape /dev/null $USER_UPDATE_FLAG_FILE
         echo "This file indicates that the Landscape client needs to send updated user information to the server." >> $USER_UPDATE_FLAG_FILE
+
+        # To work around bug #1735100 we rewrite file-local landscape sources
+        # with the trusted flag, as they have no release file, thus are
+        # unsigned repositories.
+        LANDSCAPE_INTERNAL_SOURCES=/etc/apt/sources.list.d/_landscape-internal-facade.list
+        if grep -q -e "^deb file:" $LANDSCAPE_INTERNAL_SOURCES; then
+            cp $LANDSCAPE_INTERNAL_SOURCES $LANDSCAPE_INTERNAL_SOURCES.orig
+            sed -i 's/^deb file:/deb [ trusted=yes ] file:/' $LANDSCAPE_INTERNAL_SOURCES
```

Comment:
Hrm, so do we also need to kick apt-get update here too now that we've re-written the sources file?

Reply:
landscape-client's package-monitor should take care of that once the source entry is correct. Also triggering apt from a package update triggered by apt seems like a bad plan.